cache/buntdb.go

```diff
@@ -38,9 +38,9 @@ func BuntGet(key string, w http.ResponseWriter) (cacheExist bool) {
 		val, err := tx.Get(key)
 		if err == nil {
 			cacheExist = true
-			w.Header().Set("Cache-Server", "prestd")
-			w.WriteHeader(http.StatusOK)
-			w.Write([]byte(val))
+			http.ResponseWriter.Header(w).Set("Cache-Server", "prestd")
+			http.ResponseWriter.WriteHeader(w, http.StatusOK)
+			http.ResponseWriter.Write(w, []byte(val))
```

Comment:
Since the 'http.ResponseWriter' is an interface not defined here, I believe that there is a chance that we get nil panics, isn't it?